enzy_htp/_interface/amber_interface.py

```diff
@@ -541,76 +503,58 @@ def md_run(self, prmtop: str, inpcrd: str, work_dir: str, mode: str = "CPU") ->
 
         return prod_nc
 
-    def run_tleap(self,
```

Comment:
Somehow it changes this function back. Please keep run_tleap() and discard mutate() and renumber_pdb()

Reply:
Done with commit.